etcdserver/server.go

```diff
@@ -775,6 +775,22 @@ func (s *EtcdServer) triggerSnapshot(ep *etcdProgress) {
 	ep.snapi = ep.appliedi
 }
 
+// CutPeer drops messages to the specified peer.
```

Comment:
@xiang90 Do you mean `Pause(types.ID)`? We already have `PauseSending` and `ResumeSending` methods for `EtcdServer`.

```go
func (r *raftNode) pauseSending() {
	p := r.transport.(rafthttp.Pausable)
	p.Pause()
}

func (r *raftNode) resumeSending() {
	p := r.transport.(rafthttp.Pausable)
	p.Resume()
}
```

But this doesn't pause the remotes.

```go
func (t *Transport) Pause() {
	for _, p := range t.peers {
		p.(Pausable).Pause()
	}
}

func (t *Transport) Resume() {
	for _, p := range t.peers {
		p.(Pausable).Resume()
	}
}
```